### ISSUE TITLE
Currency conversion does not work

### ISSUE THREAD

**Issue by jarun** [alexanderepstein/Bash-Snippets#173]
**Issue Label:**
* [x] Bug
* [ ] New feature
* [ ] Enhancement
* [ ] New component

**Description:**
 Conversion from USD to INR does not show correct results.

```
$ currency
What is the base currency: USD
What currency to exchange to: INR
What is the amount being exchanged: 1

=========================
| USD to INR
| Rate: 2.0
.
2.4
.18
.
| USD: 1
| INR: 2.0
2.0
2.4
.18
.18
=========================
```

Today the conversion rate is: 1 USD = 70.87 INR

### If its a bug make sure to include this section.
**OS and OS version:**
* [ ] Mac
* [ ] Linux 32 Bit
* [x] Linux 64 Bit
* [ ] Windows 32 Bit
* [ ] Windows 64 Bit

OS Version: Ubuntu 16.04

I am using master.

**Comment by alexanderepstein** [comment]
Seems like the endpoint of the api being used changed. Should be fixed now with b270518de8c96ed1e67fafd7701ae08aa932ec1e

**Comment by jarun** [comment]
It's fixed. Thanks!
